src/__tests__/role-helpers.js

```diff
@@ -23,6 +24,15 @@ function setup() {
 
   <nav data-testid='a-nav' />
   
+  <!-- div without an explicit role is given a "generic" role by default -->
+  <div aria-hidden="true" data-testid="a-hidden-div-with-a-generic-role">
+    <span>Some hidden content</span>
+  </div>
+  
+  <div aria-hidden="true" data-testid="a-hidden-div-without-a-generic-role" role="region">
```

Comment:
AFAIR, `aria-hidden` isn't the only flag that signals a hidden element, it can also be the attribute `hidden`. Maybe we should also test for that?

Reply:
Sounds good to me,
pushed it :)